README.md

```diff
@@ -257,13 +255,12 @@ disc_template =  new/%%A/%%y - %%d/%%A - %%d
 # ...
 ```
 
-## Running uninstalled
+## Developing
 
-To make it easier for developers, you can run whipper straight from the
-source checkout:
+To make it easier for developers, whipper can be installed in live mode:
 
 ```bash
-python2 -m whipper -h
+python2 -m pip install --user -e .
```

Comment:
You're changing the meaning and purpose of this section. Is this intentional? Also, you're not installing `whipper` here (as the description preceding the command claims), only its dependencies.

Reply:
It's intentional, with the changes centered on building whipper as a package it makes sense to try to use it as much as possible to depend more on the dependencies being installed by pip.

The command _does_ install the package on live mode. This means that any change in the code is instantly picked up as part of the package. This is done through symlinks, instead of the usual copying of files that happens when installing files.

This is something that would not be used in the long run, as tox would replace this workflow: it can be change to run a specific test with a specific python interpreter. I would need to check how tests are organized to make this happen, though.